i18n/pl/cosmic_settings.ftl

```diff
@@ -123,6 +123,16 @@ orientation = Kierunek
 scheduling = Harmonogram
     .manual = Ręcznie ustawiony harmonogram
 
+dialog = Dialog
+    .title = Czy Zachować Te Ustawienia Wyświetlacza?
+    .keep-changes = Zachowaj Zmiany
+    .change-prompt = Ustawienia automatycznie powrócą do poprzednich za { $time ->
+        [1] sekundę.
```

Comment:
@VandaLHJ ,
This improvement with plural forms doesn't work on my system. For some reason, after this change, the entire line is displayed in English...